pkg/shared/stopwords.go

```diff
@@ -0,0 +1,53 @@
+package shared
+
+import (
+	"slices"
+
+	"go.rtnl.ai/nlp/pkg/enum"
+)
+
+// ############################################################################
+// Stop Words Functions
+// ############################################################################
+
+// Returns true if word is a stop word for the given language. If the language
+// is not implemented this function will return false.
+func IsStopWord(word string, lang enum.Language) bool {
```

Comment:
Can we add a `strings.Lower` here to ensure that the comparison is case insensitive here?